README.md

```diff
@@ -66,7 +69,7 @@ export default FooPage;
 
 Note that we import the compiled CSS `@material/foo/dist/mdc.foo` so we don't have to recompile Sass files.
 
-3. Add a 24px icon associated with the component (e.g. `ic_foo_24px.svg`) to the `src/images` directory.
+3. Add a SVG image associated with the component (e.g. `ic_foo_180px.svg`) to the `src/images` directory.
```

Comment:
Nit: most of our images don't have the `ic_` prefix anymore so that can be removed. Also in the next step.